src/identity/hooks/useAvatar.test.tsx

```diff
@@ -3,63 +3,61 @@
  */
 
 import { renderHook, waitFor } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { publicClient } from '../../network/client';
 import { useAvatar, ensAvatarAction } from './useAvatar';
-import { useOnchainActionWithCache } from '../../utils/hooks/useOnchainActionWithCache';
 
 jest.mock('../../network/client');
-jest.mock('../../utils/hooks/useOnchainActionWithCache');
 
 describe('useAvatar', () => {
   const mockGetEnsAvatar = publicClient.getEnsAvatar as jest.Mock;
-  const mockUseOnchainActionWithCache = useOnchainActionWithCache as jest.Mock;
 
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
   it('returns the correct ENS avatar and loading state', async () => {
+    const queryClient = new QueryClient();
+    function ReactQueryTestProvider({ children }: { children: React.ReactNode }) {
```

Comment:
Should `ReactQueryTestProvider` be a test utility maybe? Just in this PR was created 4 times.